source/geod24/LocalRest.d

```diff
@@ -1373,11 +1395,11 @@ unittest
     assert(current4 - current2 >= 1.seconds);
 
     // Now drop many messages
-    n1.sleep(1.seconds, true);
-    for (size_t i = 0; i < 500; i++)
+    n1.sleep(3.seconds, true);
+    for (size_t i = 0; i < 100; i++)
```

Comment:
Why are those values changed ? It looks like the new implementation is much slower ?

Reply:
Ping. Why were these values changed?

Reply:
Ping

Reply:
This code is intended to delete incoming messages for one second. I changed it because I couldn't process all 500 in a second.